src/components/admin/Dropdown/Dropdown.module.scss

```diff
@@ -0,0 +1,16 @@
+@use "scss/attendee/theme";
+
+// @debt: we might need another set of variables for admin, just like we have for attendee
+// to cover cases when tailwind classes are not enough
```

Comment:
> to cover cases when tailwind classes are not enough

Well... ☹️ 

[OOS]
It is not as much as having a set of constants, but having separation between the Tailwind "world" and the `.module.scss` one. 

This might put the problem in perspective: if user clicks links from admin that take them to attendee side and from attendee to admin side, both systems shouldn't end up loaded in the same DOM.

And in this case, users don't even need to navigate between admin/attendee sides, they buy one, get one free 🤪 

👉 This debt should be repaid soon, not forgotten❗

Reply:
[Q]
@alexey-petrov can we have a copy-paste duplicates of this Dropdown for admin and attendee sides but removing the extra styles?

If it's something simple, we can do a follow up PR to divorce the uses.

Reply:
this dropdown is not used at the `attendee` side at all. I've removed this `attendee` usage here and edited the debt comment saying that we need another set of admin side variables added in the future
for now, this style will be hardcoded as `white` instead of using attendee variable
[1ae2f8b](https://github.com/sparkletown/sparkle/pull/2944/commits/1ae2f8b185da5056b37e5478d84f29d2daf6cce8)